fbw-a380x/src/systems/systems-host/systems/LegacyFuel.ts

```diff
@@ -0,0 +1,431 @@
+import { MathUtils } from '@flybywiresim/fbw-sdk';
+import {
+  ConsumerSubject,
+  EventBus,
+  GameStateProvider,
+  Instrument,
+  KeyEventManager,
+  Wait,
+  WeightBalanceEvents,
+} from '@microsoft/msfs-sdk';
+import { FuelSystemEvents } from 'systems-host/systems/FuelSystemPublisher';
+
+/**
+ * This is needed to initialize the desired fuel L:Var on load to sync with the fuel quantity as per the fuel state management per ATC ID
+ * This is a temporary solution until all fuel state related ops are contained in the same module.
+ */
+
+/* TODO: remove this file after proper FQMS is implemented in Rust */
+export class LegacyFuel implements Instrument {
+  private readonly NUMBER_OF_TRIGGERS = 42;
```

Comment:
Nitpick: I believe it's better to make those static and then reference them like `NUMBER_OF_TRIGGERS` (wthout `this`).

Reply:
I honestly forgot I could do that :D